app/experimenter/experiments/models.py

```diff
@@ -437,7 +437,8 @@ def computed_enrollment_days(self):
         paused_changelogs = [
             c
             for c in changes
-            if c.experiment_data["is_paused"]
+            if "is_paused" in c.experiment_data
+            and c.experiment_data["is_paused"]
```

Comment:
Aren't these redundant?

Reply:
Negative, the 500 error in prod is happening because that key may not always be present in which case it will throw a `KeyError` so you need to check that the key is present first, and because Python short circuits boolean operations (properly I might add, looking at you JEXL), it won't continue evaluating the following clauses is the first one fails.